crates/egui/src/context.rs

```diff
@@ -1060,12 +1060,12 @@ impl Context {
             // but also to know when we have reached the widget we are checking for cover.
             viewport.this_frame.widgets.insert(w.layer_id, w);
 
-            if w.sense.focusable {
+            if w.sense.focusable && !ignore_focus {
```

Comment:
Could we just set `w.sense.focusable = false` instead of having an extra parameter?